modules/microshift-ops-config-embed-ostree.adoc

```diff
@@ -0,0 +1,162 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-apply-ops-ostree-offline-use_{context}"]
+= Applying catalogs and Operators in an offline RHEL for Edge image
+
+After you have created a {op-system-ostree} image for offline use and configured {microshift-short} networking settings for offline use, you can configure the namespace and edit the custom resources (CR) generated by the oc mirror dry run for running your Operators.
+
+.Prerequisites
+* You have a {op-system-ostree} image.
+* Networking is configured for offline use.
+* You completed the oc mirror dry run procedure.
+
+.Procedure
+
+. Create a `CatalogSource` custom resource (CR) by using the following example YAML:
++
+[source,terminal]
+----
+$ cat > <my-catalog-source-cr.yaml> <1>
+----
+<1> Replace _<my-catalog-source-cr.yaml>_ with the name of your Catalog Source CR.
++
+.Example `CatalogSource` YAML
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cs-redhat-operator-index
+  namespace: openshift-marketplace <1>
+spec:
+  image: registry.example.com/redhat/redhat-operator-index:v4.15
+  sourceType: grpc
+  displayName:
+  publisher:
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> The global namespace. Setting the `metadata.namespace` to `openshift-marketplace` enables the catalog to run in all namespaces. Subscriptions in any namespace can reference catalogs created in the `openshift-marketplace` namespace.
++
+[NOTE]
+====
+The default namespace security for `openshift-marketplace` is `baseline`, therefore a catalog source custom resource (CR) created in that namespace does not require a `spec.grpcPodConfig.securityContextConfig` value to be set. You can set a `legacy` or `restricted` value if required for the namespace and Operators you want to use.
```

Comment:
I don't think "namespace security" is the correct term. See: https://docs.openshift.com/container-platform/4.15/authentication/understanding-and-managing-pod-security-admission.html